### ISSUE TITLE
Medomai the Ageless bug when cheated in play?

### ISSUE THREAD

**Issue by drmDev** [magefree/mage#2989]
I had a Medomai the Ageless in play and I attacked and counnected with it and got an extra turn. At the end of that first turn I sacrificed it using Altar of Bone. Next turn during beginning of combat I cast the last card in my hand, Cauldron Dance, to return it to the battlefield tapped and was able to attack with it (even though this was my second turn in a row). I had no cards in hand so I didn't put another creature in play. The Medomai did connect and I had just begun my 3rd turn in a row when my opponent conceded, so i don't know if it would have worked again or not.

tl;dr: Cheated in Medomai the Ageless on an extra turn and was able to attack with it and get ANOTHER extra turn.

(from forums)

**Comment by drmDev** [comment]
Paging @emerald000  :) Is the "Medomai the Ageless cannot attack during extra turns" part only relevant to that instance of Medomai? In the case here where it was sacrificed and rebirthed into play with haste, should it be able to attack again and grant another extra turn?

**Comment by LevelX2** [comment]
IMHO: The "can't attack during extra turns" is relevant for all instances of Medomai.

9/15/2013 	An “extra turn” is any turn created by a spell or ability. Notably, it doesn’t include additional turns taken in tournaments after time expires for a round.
9/15/2013 	It doesn’t matter whose turn it is. In very rare cases, an opponent may be taking an extra turn and gain control of Medomai. Medomai can’t attack during that turn.

**Comment by drmDev** [comment]
Unable to duplicate issue with UT. Double check if everything looks right in the tests. Will need to test manually to confirm resolution.

**Comment by emerald000** [comment]
If we are in an extra turn, for whatever reason, Medomai can't attack.

**Comment by jeffwadsworth** [comment]
Just tested this multiple times.  It works perfectly.  I could not attack during the extra turn.

**Comment by jeffwadsworth** [comment]
![worksfinehere](https://cloud.githubusercontent.com/assets/1496996/24085174/fba5976c-0cc4-11e7-8443-15b6cc9829d5.jpg)

**Comment by drmDev** [comment]
manual and automated testing cannot duplicate the issue. closing.
